support-frontend/app/controllers/DigitalSubscription.scala

```diff
@@ -44,9 +47,9 @@ class DigitalSubscription(
   def digital(countryCode: String): Action[AnyContent] = CachedAction() { implicit request =>
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
     val title = "Support the Guardian | Digital Pack Subscription"
-    val id = "digital-subscription-landing-page-" + countryCode
-    val js = "digitalSubscriptionLandingPage.js"
-    val css = "digitalSubscriptionLandingPage.css"
+    val id = EmptyDiv("digital-subscription-landing-page-" + countryCode)
```

Comment:
I'm not sure that this type is appropriate here. `id` IS a string not a div

Reply:
Ah I see why this is now, I think it would be good to rename the `id` parameter as it doesn't describe what is passed in any more

Reply:
thanks for the commment Rupert.

the ID is a string, but the type is representing an empty div, which only has an ID.
the alternative is SSRContent which has an id and some HTML.

I did think about having an ID value class that wraps the string, and an EmptyDiv that represents the sum ADT, but I am also conscious about going too deep too quickly.

So this is kind of a combination value class and sum ADT (sealed trait)

Given the current intent, I prefer a name to represent the meaning, but I can split into two or rename to something more clear along those lines?  What do you reckon?

Reply:
ah my comment crossed over yours, yes I'll rename ID throughout to `mainElement`(or other idea?)